### ISSUE TITLE
Regardless of Ngrock or local tunnel or gradio, there seems to be an issue with the python again. 

### ISSUE THREAD

**Issue by SDPDVADER** [TheLastBen/fast-stable-diffusion#1914]
/usr/local/lib/python3.9/dist-packages/tensorflow_io/python/ops/__init__.py:98: UserWarning: unable to load libtensorflow_io_plugins.so: unable to open file: libtensorflow_io_plugins.so, from paths: ['/usr/local/lib/python3.9/dist-packages/tensorflow_io/python/ops/libtensorflow_io_plugins.so']
caused by: ['/usr/local/lib/python3.9/dist-packages/tensorflow_io/python/ops/libtensorflow_io_plugins.so: undefined symbol: _ZN3tsl6StatusC1EN10tensorflow5error4CodeESt17basic_string_viewIcSt11char_traitsIcEENS_14SourceLocationE']
  warnings.warn(f"unable to load libtensorflow_io_plugins.so: {e}")
/usr/local/lib/python3.9/dist-packages/tensorflow_io/python/ops/__init__.py:104: UserWarning: file system plugins are not loaded: unable to open file: libtensorflow_io.so, from paths: ['/usr/local/lib/python3.9/dist-packages/tensorflow_io/python/ops/libtensorflow_io.so']
caused by: ['/usr/local/lib/python3.9/dist-packages/tensorflow_io/python/ops/libtensorflow_io.so: undefined symbol: _ZTVN10tensorflow13GcsFileSystemE']
  warnings.warn(f"file system plugins are not loaded: {e}")

**Comment by kieranxsomer** [comment]
Yea we're all having this issue, and one suggestion was to clear teh plugins folder, but that ONLY WORKED so far, and kdiffusion unloads itself. I'm gonna try a reinstall

**Comment by DanoGL** [comment]
Yeah I have the same issue too, I don’t know what to do, it doesn’t affect image generation (at least with the little trying I did), but I can’t use img2img and the interrogate deepboru options

**Comment by TheLastBen** [comment]
that's a colab thing, not caused by the notebook, I'll investigate it

**Comment by DanoGL** [comment]
> 

Thank you Ben, greetings from 🇲🇽
